.jshintrc

```diff
@@ -30,6 +30,7 @@
     "$" : false,
     "jasmine" : false,
     "$$": false,
-    "expect" : false
+    "expect" : false,
+    "xit" : false
```

Comment:
this can lead to many problems, please don't enable xit but comment the tests that are irrelevant instead.